alodataset/coco_detection_dataset.py

```diff
@@ -282,15 +298,14 @@ def show_random_frame(coco_loader):
 def main():
     """Main"""
     logging.basicConfig(
-        level=logging.INFO,
-        format="[%(asctime)s][%(levelname)s] %(message)s",
-        datefmt="%d-%m-%y %H:%M:%S",
+        level=logging.INFO, format="[%(asctime)s][%(levelname)s] %(message)s", datefmt="%d-%m-%y %H:%M:%S",
```

Comment:
Its getting weird to use logger in this main() method while we don't use it in other main methods in the datasets folder. I would suggest to remove it from here, removing the print, and use the sample as for the others datasets.

Reply:
Done